backend/src/services/auth.rs

```diff
@@ -37,10 +39,31 @@ impl AuthService {
             .map_err(HashPasswordFailed)?
             .to_string();
 
-        let uuid = Uuid::new_v4();
-        users::Model::create_with_password(database, uuid, password_hash, &credentials).await?;
+        let confirmation_code =
+            users::Model::create_with_password(&app_state.database, password_hash, &credentials)
+                .await?;
+        let mut confirmation_link = "".to_string();
+        confirmation_link.push_str(&EnvConfig::get().email_confirm_backend_url.clone());
+        confirmation_link.push('/');
+        confirmation_link.push_str(&confirmation_code);
+
+        Email {
+            sender: (
+                Some("Hack4Krak Authentication".to_string()),
+                "auth@hack4krak.pl".to_string(),
+            ),
+            recipients: vec![credentials.email],
+            subject: "Potwierdzenie rejestracji".to_string(),
+            template: EmailTemplate::EmailConfirmation,
```

Comment:
I don't like that you define some stuff in `EmailTemplate` (possible placeholders), but some stuff here (title). Also there is no type safety for placeholders etc, but this is topic for separate issue